src/annotator/adder.html

```diff
@@ -3,8 +3,8 @@
     <button class="annotator-adder-actions__button h-icon-annotate js-annotate-btn">
       <span class="annotator-adder-actions__label" data-action="comment">Annotate</span>
     </button>
-    <button class="annotator-adder-actions__button h-icon-highlight js-highlight-btn">
+    <!-- <button class="annotator-adder-actions__button h-icon-highlight js-highlight-btn">
```

Comment:
Instead of commenting this out, can we use a feature flag to make the class hide it?

Reply:
I added a new commit to show that it's doable and how I did. Let me know what you think. 

However, I didn't understand the value of having a flag rather than commenting them out. Is it for keeping the options available just in case we might want to have this highlight button in the future? 

Is your comment specific to this element or is it general?

Reply:
Was a general comment, if we use feature flags, we can commit upstream. I think there's good value in providing these as options.